tests/Resources/ODS.robot

```diff
@@ -400,3 +400,30 @@ Delete RHODS Config Map
     [Arguments]     ${name}  ${namespace}=redhat-ods-applications
     OpenShiftLibrary.Oc Delete    kind=ConfigMap  name=${name}  namespace=${namespace}
 
+Wait Until Cuda Builds Are Completed
+    [Documentation]    Waits until pytorch and tensorflow builds have Complete Status.
+    ...    The keyword waits 20 minutes for each build to be in that status, failing otherwise.
```

Comment:
Is 20 minutes enough?
What about all the cuda builds before pytorch/tensorflow? I think this should wait until either build at least starts, and then eventually wait for 20 minutes until the build is completed.
Otherwise if we run this TC right after install it will fail.

Reply:
+1 it should wait for all the cuda builds to be finished, including the base ones before notebook images.

if 20 minutes is enough? I guess it is enough because the robot wait 20 minutes for each build, which take max 10 minutes, if I'm not wrong

Reply:
In the end the test is using "Wait Until All Builds Are Complete", which will wait 20 minutes for each build. As @bdattoma  said, they all take aprox 10 minutes, so it should be enough